kola/tests/misc/nfs.go

```diff
@@ -48,17 +48,6 @@ systemd:
 )
 
 func init() {
-	register.Register(&register.Test{
-		Run:         NFSv3,
-		ClusterSize: 0,
-		Name:        "linux.nfs.v3",
-		Distros:     []string{"cl"},
```

Comment:
Is there a reason to not test this on fcos?

Reply:
FCOS just ships the client, not nfs-server (https://github.com/coreos/fedora-coreos-tracker/issues/121)